src/spring/azext_spring/application_live_view.py

```diff
@@ -0,0 +1,69 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=unused-argument, logging-format-interpolation, protected-access, wrong-import-order, too-many-lines
+from knack.log import get_logger
+
+from .vendored_sdks.appplatform.v2022_11_01_preview import models
+from .dev_tool_portal import (try_get_updatable as get_dev_tool_portal,
+                              create_or_update as create_or_update_dev_tool_portal,
+                              _get_desired_state as get_dev_tool_portal_desired_state)
+from ._utils import (wait_till_end)
+
+DEFAULT_NAME = "default"
+
+logger = get_logger(__name__)
+
+
+def show(cmd, client, service, resource_group):
+    return client.application_live_views.get(resource_group, service, DEFAULT_NAME)
+
+
+def create(cmd, client, service, resource_group, no_wait=False):
+    poller = client.application_live_views.begin_create_or_update(resource_group, service, DEFAULT_NAME,
+                                                                  models.ApplicationLiveViewResource())
+    dev_tool_portal_poller = _get_enable_dev_tool_portal_poller(cmd, client, service, resource_group)
+    pollers = [x for x in [poller, dev_tool_portal_poller] if x is not None]
+    if not no_wait:
+        wait_till_end(cmd, *pollers)
```

Comment:
Just out of curiosity, may I ask why don't you use `sdk_no_wait`?

Reply:
There may be 2 pollers to wait

Reply:
Got it, thanks~